### PR TITLE
Update version number for Lua updatePolicy

### PR DESCRIPTION
**Pull request by cmouse**
### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled and tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added regression tests
- [ ] added unit tests

### CODE REVIEW

**Comment by rgacogne** [comment]
If I'm not mistaken, the Lua update policy feature has been merged in master and not backported to 4.0.x, so unless we decide otherwise it should probably be "Since 4.1.0".

**Comment by Habbie** [comment]
Nit: it's Lua, not LUA

**Comment by Habbie** [comment]
Indeed, it should say 4.1.0.

File: docs/markdown/authoritative/dnsupdate.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ This is a short description of how DNS update messages are processed by PowerDNS
 
 # Update policy
 
-Since 4.0.0, you can define a Lua script to handle DNS UPDATE message authorization.
+Since 4.0.2, you can define a Lua script to handle DNS UPDATE message authorization.
```
Comment (Habbie) [inline docs/markdown/authoritative/dnsupdate.md]: Please change to 4.1.0.

**Comment by cmouse** [comment]
Nits fixed. Sorry it took so long.

**Pull request merged**